dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/security/config/WebMvcConfig.java

```diff
@@ -74,12 +80,9 @@
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
+import com.google.common.collect.ImmutableMap;
 
-import static org.springframework.http.MediaType.parseMediaType;
+import lombok.extern.slf4j.Slf4j;
```

Comment:
Could this be removed?

Reply:
I see @Slf4j is annotated, but it is not in use, we should propbably remove it.

Reply:
This was part of the original file, it was just reordered.
But we do not have anyone using it in the class actually, and it can be removed for sure
Good catch :+1:

Reply:
Done